shared/util/phone-numbers/index.tsx

```diff
@@ -39,4 +39,10 @@ export const validateNumber = (rawNumber: string, region: string) => {
   }
 }
 
+export const formatPhoneNumber = (rawNumber: string) => {
+  // TODO: support non-US numbers
```

Comment:
here: 
https://github.com/keybase/client/blob/f85e66bde8535a3ec34d3aa021e1f2f3171b7bc1/shared/util/phone-numbers/index.tsx#L49-L59

Reply:
The design uses `+1 (234) 555-6789` whereas this returns either `+12345556789` or `+1 234-555-6789`, I forget which.

Reply:
Have you tried out `NATIONAL`? I only know its there. Otherwise you could hack around it by using the type as you go formatter.

Reply:
@pzduniak the current code uses `NATIONAL`.